mne/time_frequency/csd.py

```diff
@@ -255,3 +221,187 @@ def compute_epochs_csd(epochs, mode='multitaper', fmin=0, fmax=np.inf,
                                              frequencies=frequencies[i],
                                              n_fft=n_fft))
         return csds
+
+
+@verbose
+def csd_array(X, sfreq, mode='multitaper', fmin=0, fmax=np.inf,
+              fsum=True, n_fft=None, mt_bandwidth=None,
+              mt_adaptive=False, mt_low_bias=True, verbose=None):
+    """Estimate cross-spectral density from an array.
+
+    .. note:: Results are scaled by sampling frequency for compatibility with
+              Matlab.
+
+    Parameters
+    ----------
+    X : array-like, shape (n_replicates, n_series, n_times)
```

Comment:
we should stick to standard names we use everywhere in the code base. Please at least rename n_series to n_channels